plugins/auth-auth0/README.md

```diff
@@ -0,0 +1,228 @@
+      "accessToken": "{ACCESS_TOKEN}",
+      "actionName": "Add user details to access token",
+      "clientName": "Custom SPA",
+      "apiName": "Custom API",
+      "audience": "http://example.com"
+    }
+    // General settings ( See General Configuration )
+  }
+}
+```
+
+### Manually
+
+This method requires the user to manually create the required values in the Auth0 account and provide the values in the plugin settings.
+
+#### Method Specific Configuration
+
+This method requires the following values to be provided in the plugin settings.
+
+- `useManagementApi` : `false` ( To use this method )
+- `domain` : The domain of the Auth0 application (client) ( See [copy domain](#L179) )
+- `clientID` : The client id of the Auth0 application (client) ( See [copy client id](#L180) )
+- `audience` : The audience/identifier of the API which you have created in the Auth0 account. ( See [copy audience](#L188) )
+- `issuerURL` : The issuer base url of the API which you have created in the Auth0 account. ( See [copy issuer base url](#L189) )
+
+**Example**
+
+```json
+{
+  "settings": {
+    "useManagementApi": false,
+    "domain": "dev-p27ryta7rrcalcea.us.auth0.com",
+    "clientID": "A9Dvb0BS8His5lrzRNduNxtGcoTHagid",
+    "audience": "https://sample-nest.demo.com",
+    "issuerURL": "https://dev-p27ryta7rrcalcea.us.auth0.com/"
+    // General settings ( See General Configuration )
+  }
+}
+```
+
+## Elaboration
+
+This section elaborates on the steps to be followed to get the required values for the plugin settings. Prerequisite for this is to have an Auth0 account.
+
+### Get Management API Token
+
+- Go to [Management Explorer](https://manage.auth0.com/#/apis/management/explorer) and copy the token as shown there.
+  If there is no token, click on the `Create Testing Application` button and create a testing application. Then copy the token. This is the value of the `accessToken` field in the plugin settings.
+
+- Go to [Management Settings Page](https://manage.auth0.com/#/apis/management/settings) and copy the value of the `Identifier` field. This is the value of the `identifier` field in the plugin settings.
+
+### Create an Auth0 application
+
+- Go to [Applications Page](https://manage.auth0.com/#/applications) and create a new application of type `Single Page Web Applications`.
+- Select React in the Quickstart tab or follow the steps below.
+- Change the following values to their keys :-
+
+  - Allowed Callback URLs : http://localhost:3001/auth-callback
+  - Allowed Logout URLs : http://localhost:3001/login
+  - Allowed Web Origins : http://localhost:3001
+
+- Go to the settings tab and copy the values of the following :-
+
+  - Domain
+  - Client ID
+
+  ![Auth0 Application Settings](./assets/auth0-admin-settings.png)
+
+### Create an Auth0 API
+
+- Go to the [API Dashboard](https://manage.auth0.com/#/apis) and create a new API with the following values :-
+
+  - Identifier : https://sample-nest.demo.com
```

Comment:
```suggestion
  - Identifier : http://localhost:3001
```

as you show in the example here: https://github.com/amplication/plugins/pull/177/files#diff-56ba0222f9cc8ff2d6c06bcef99f053c65035b5823d83eeb60f1bf8532a608e0R200